src/components/PluginSettings/index.tsx

```diff
@@ -243,6 +245,26 @@ export default ErrorBoundary.wrap(function Settings() {
         );
     };
 
+    const [newPlugins, , newPluginsLoading] = useAwaiter(() => DataStore.get("Vencord_existingPlugins").then((cachedPlugins: Record<string, number>) => {
+        const dateNow: number = Date.now() / 1000;
+        const existingPlugins: Record<string, number> = {};
+        const sortedPluginNames = Object.values(sortedPlugins).map(plugin => plugin.name);
+
+        let newPlugins: Array<string> = [];
+        sortedPlugins.forEach(plugin => {
+            existingPlugins[plugin.name] = cachedPlugins[plugin.name] ?? dateNow;
```

Comment:
```suggestion
            existingPlugins[plugin.name] = cachedPlugins?.[plugin.name] ?? dateNow;
```

when indexing a nullable value, you can simply use `?.` before the brackets to ensure that it won't error

Reply:
since this is the only usage of `cachedPlugins` that I can see within this function, this should be enough to ensure no error happens

Reply:
(yes I know, this is weird syntax, but oh well)